File: eShelvesAPI/eShelvesAPI/Models/Knjiga.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace eShelvesAPI.Models
{
	public class Knjiga
	{
		public int Id { get; set; }
		public string Naslov { get; set; }
		public string Opis { get; set; }
		public string ISBN { get; set; }
		public DateTime Objavljena { get; set; }

		public int AutorId { get; set; }
		public Autor Autor { get; set; }

		public List<Ocjena> Ocjenas { get; set; }
		public List<Polica> Policas { get; set; }
		public List<Kategorija> Kategorijas { get; set; }
		//slika
	}
}

File: eShelvesAndroid/app/src/main/java/com/readmore/tonka/models/Knjiga.java
package com.readmore.tonka.models;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by anton_000 on 30.4.2015..
 */
public class Knjiga implements Serializable{
    public int Id;
    public String Naslov;
    public String Opis;
    public String ISBN;
    public int AutorID;
    public String NazivAutora;
}
